### PR TITLE
docs(components): Add documentation landing page - FRONT-1828

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx [hunk 1]
```diff
@@ -6,13 +6,4 @@ order: 2
...
-  system="ec"
-  selectedKind="components-navigation-inpage-navigation"
-  selectedStory="default"
-  frameHeight="350"
-  disableAutoResize={true}
-  showFrame
->
-  <Html markup={inpageNavigationExample} />
-</Playground>
+<Html markup={inpageNavigationExample} />
```
Comment (kalinchernev) [inline src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx]: Why this change? I don't mind having the html directly on the page if it works showing the spy behavior.

**Review by kalinchernev — CHANGES_REQUESTED** [review]
The landing page is ok, but the inpage navigation's scrollspy in the demo is not quite as good

File: src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx (hunk 1 above)
Comment (papegaill) [reply src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx]: Ouups !! Thks @kalinchernev this is a complete mistake, thank you for noticing it !!

**Review by kalinchernev — APPROVED** [review]
